tide_core.info.yml

```diff
@@ -50,7 +50,6 @@ dependencies:
   - fast_404:fast404
   - link_field_autocomplete_filter:link_field_autocomplete_filter
   - linkit:linkit
-  - login_redirect_per_role:login_redirect_per_role
```

Comment:
I don't think we need `login_redirect_per_role` - it conflicts with Simple OAuth and breaks preview redirects after user login. All roles also redirect to `/admin/workbench` after login so we're not using the module's main feature anyway.